lib/Auth/Process/DatabaseCommand.php

```diff
@@ -43,41 +43,47 @@ public static function insertLogin(&$request, &$date)
         $month = $date->format('m');
         $day = $date->format('d');
 
-        if (is_null($idpEntityID) || empty($idpEntityID) || is_null($spEntityId) || empty($spEntityId)) {
+        if ($idpEntityID === null || empty($idpEntityID) || $spEntityId === null || empty($spEntityId)) {
             Logger::error(
-                "Some from attribute: 'idpEntityId', 'idpName', 'spEntityId' and 'spName'" .
-                " is null or empty and login log wasn't inserted into the database."
+                'Some from attribute: \'idpEntityId\', \'idpName\', \'spEntityId\' and \'spName\'' .
```

Comment:
I personally would prefer if the parameters in the list would be wrapped in double quotes, thus we will get rid of the backslash. @pajavyskocil @BaranekD - opinions?